### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#2770] [General]
# Roadmap

This is the roadmap for Meeting Media Manager. This discussion shows the development progress and timeline.

## Released

- See [release notes](https://github.com/sircharlo/meeting-media-manager/releases), [changelog](https://github.com/sircharlo/meeting-media-manager/blob/master/CHANGELOG.md) or [issues](https://github.com/sircharlo/meeting-media-manager/issues?q=is%3Aissue+label%3Aimplemented+) and [discussions](https://github.com/sircharlo/meeting-media-manager/discussions?discussions_q=label%3Aimplemented+) labeled https://github.com/sircharlo/meeting-media-manager/labels/implemented.

## Will be released in the next version

See [issues](https://github.com/sircharlo/meeting-media-manager/issues?q=is%3Aissue+label%3Aawaiting-release) and [discussions](https://github.com/sircharlo/meeting-media-manager/discussions?discussions_q=is%3Aopen+label%3Aawaiting-release) labeled https://github.com/sircharlo/meeting-media-manager/labels/awaiting-release.

## In development

See [issues](https://github.com/sircharlo/meeting-media-manager/issues?q=is%3Aopen+is%3Aissue+label%3Ain-progress) and [discussions](https://github.com/sircharlo/meeting-media-manager/discussions?discussions_q=is%3Aopen+label%3Ain-progress) labeled https://github.com/sircharlo/meeting-media-manager/labels/in-progress.

##  To be developed in the future

- #2781

## Questions?

If you have any questions after reading this, please feel free to ask them by creating a [new discussion](https://github.com/sircharlo/meeting-media-manager/discussions/new/choose).

**Comment by sircharlo** [comment]
See https://github.com/sircharlo/meeting-media-manager/discussions/5260
